scripts/mysqld_safe.sh

```diff
@@ -258,6 +258,16 @@ add_mysqld_ld_preload() {
   lib_to_add="$1"
   log_notice "Adding '$lib_to_add' to LD_PRELOAD for mysqld"
 
+  # Check if the library is in the reduced number of standard system directories
+  case "$lib_to_add" in
+    /usr/lib64* | /usr/lib/x86_64-linux-gnu* | /usr/lib* | ${MY_BASEDIR_VERSION}/lib*)
```

Comment:
may be we can simplified this `in`
/usr/lib* | ${MY_BASEDIR_VERSION}/lib* should be enough
as * would include all other paths